src/ripple/overlay/Overlay.h

```diff
@@ -70,6 +70,16 @@ class Overlay
         bool expire = false;
         beast::IP::Address public_ip;
         int ipLimit = 0;
+
+        enum CrawlOptions {
```

Comment:
Just noting a micro-nit: these values are never exposed via the `Overlay` API, so they could just be migrated into `OverlayImpl`. _No change is necessary._

Reply:
That's true, I was going to use the pattern of bit flags in an enum, but realized that's a Microsoft specific quirk: [msdn.com](https://blogs.msdn.microsoft.com/vcblog/2008/03/10/visual-studio-2008-enum-bit-flags-visualization/).